tests/utils/test_logging_tracing.py

```diff
@@ -0,0 +1,17 @@
+import logging
+import unittest
+
+from opentelemetry import trace
+
+from ado_asana_sync.utils.logging_tracing import setup_logging_and_tracing
+
+
+class TestLoggingTracing(unittest.TestCase):
+    def test_setup_logging_and_tracing(self):
+        logger, tracer = setup_logging_and_tracing(__name__)
+        self.assertIsInstance(logger, logging.Logger)
+        self.assertIsInstance(tracer, trace.Tracer)
```

Comment:
**Suggestion:** The `test_setup_logging_and_tracing` test could be improved by adding more assertions. This would make the test more thorough and reliable.
```suggestion
# Suggestion: Add more assertions to the `test_setup_logging_and_tracing` test
```